packages/graphback-codegen-resolvers/src/templates/LayeredResolverTemplates.ts

```diff
@@ -0,0 +1,111 @@
+// TODO subscription support
+// TODO support for the database type instead of hardcoded lowercase
+export const createTemplate = (fieldName: string, typeName: string, subscription?: boolean): string => {
+  return `${fieldName}: (_, args, context) => {
+      validateContext(context)
```

Comment:
Is there some way this can be done before this function is executed? I don't like it here as it doesn't seem like the  resolver's responsibility. I couldn't think of how to do this though.

Reply:
Good idea: Yes.. we can do root resolver or wrap every resolver with it.. 
Generaly this code is repeated everywhere as pattern for adding more stuff in future. 

IMHO we should move this check to runtime and call it something like - validateRuntimeContext. 

As for having a resolver layer for checks.
I think we can simply have this check in service (runtime) for the moment and remove it completely?